web/src/user_topics.ts

```diff
@@ -15,7 +16,32 @@ import * as timerender from "./timerender";
 import * as ui_report from "./ui_report";
 import {get_time_from_date_muted} from "./util";
 
-const all_user_topics = new Map();
+type ServerUserTopic = z.infer<typeof user_topic_schema>;
+
+export type UserTopic = {
+    stream_id: number;
+    stream: string | undefined;
```

Comment:
So I think the caller of `get_user_topics_for_visibility_policy` really wants the streams to actually exist.

I think we should consider a small refactor to allow us to remove this type by having `get_user_topics_for_visibility_policy` return a `ServerUserTopic` and having the one caller be responsible for fetching the stream names if wanted for display purposes.